java/docker-build/Dockerfile

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+###
+# Build the image for spark-rapids-jni development environment.
+#
+# Arguments: CUDA_VERSION=[12.X.Y], OS_RELEASE=[8, 9], TARGETPLATFORM=[linux/amd64, linux/arm64]
+#
+###
+ARG CUDA_VERSION=12.9.1
+ARG OS_RELEASE=9
```

Comment:
Why is 8 / 9 an option here? I would have thought we would always build on 8

Reply:
Per discussion, I'll retain support to use 8 and 9.  Thank you for clarifying, @robertmaynard.